app/forms/openid_connect_authorize_form.rb

```diff
@@ -255,4 +265,9 @@ def validate_privileges
       )
     end
   end
+
+  def highest_level_aal(aal_values)
+    AALS_BY_PRIORITY.find { |aal| aal_values.include?(aal) } ||
+      Saml::Idp::Constants::AAL1_AUTHN_CONTEXT_CLASSREF
```

Comment:
Testing this a little bit, and I think we'll want to drop the default here since it causes `#aal` to return a string rather than an Integer.

```suggestion
    AALS_BY_PRIORITY.find { |aal| aal_values.include?(aal) }
```